wazo_calld_client/commands/calls.py

```diff
@@ -99,10 +99,10 @@ def hangup_from_user(self, call_id):
         if r.status_code != 204:
             self.raise_from_response(r)
 
-    def connect_user(self, call_id, user_id):
+    def connect_user(self, call_id, user_id, **kwargs):
```

Comment:
Could you also update the "doc", i.e. the README, to show how to pass arguments to the endpoint, please?